src/database/schemas.ts

```diff
@@ -16,7 +16,7 @@ const typeMap = {
 };
 
 export default class Database {
-  static async getObjects(schema: string, type: SQLType, details: PageData = {}): Promise<BasicSQLObject[]> {
+  static async getObjects(schema: string, type: SQLType | string, details: PageData = {}): Promise<BasicSQLObject[]> {
```

Comment:
`type` parameter should not have the additional `string` type here. Calling methods should only support `SQLType`